sanity/SanityActions.js

```diff
@@ -57,7 +57,12 @@ export class SanityActions {
   }
 
   logErrors(faultyAssets) {
-    log.error(faultyAssets.map(({ error }) => error).join('\n'))
+    log.error(
+      faultyAssets
+        .filter(({ error }) => !!error)
+        .map(({ error }) => error)
+        .join('\n')
+    )
```

Comment:
I think now that we're doing a double loop (filter+map) your original suggestion is a better option:

```typescript
for (const ({error}) of faultyAssets) {
  if (error) {
    log.error(error)
  }
}
```

but it's ok the way it is also

Reply:
Yeah, I will keep as it is, we are not going to have a lot of errors.